### PR TITLE
[PRD-461] Add descriptions

### PR DESCRIPTION
**Pull request by paultranvan**
```
### ✨ Features

* Add descriptions for store

### 🐛 Bug Fixes

*

### 🔧 Tech

*
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ Pull requests are the primary mechanism we use to change Cozy. GitHub itself has
 Fork the project on GitHub and [check out your copy locally][forking].
 
 ```
-$ git clone github.com/cozy/ogma.git
-$ cd ogma
-$ git remote add fork git://github.com/yourusername/ogma.git
+$ git clone github.com/cozy/Papillon.git
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
$ git clone github.com/cozy/cozy-papillon.git
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Pull requests are the primary mechanism we use to change Cozy. GitHub itself has
...
+$ cd Papillon
+$ git remote add fork git://github.com/yourusername/Papillon.git
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
$ git remote add fork git://github.com/yourusername/cozy-papillon.git
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ Well, we think you know how to do that. Just be sure to follow the coding guidel
 Don't forget to add tests and be sure they are green:
 
 ```
-$ cd ogma
+$ cd Papillon
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
$ cd cozy-papillon
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ $ git rebase origin/master my-branch
 $ git push -u fork my-branch
 ```
 
-Go to https://github.com/yourusername/ogma and select your branch. Click the 'Pull Request' button and fill out the form.
+Go to https://github.com/yourusername/Papillon and select your branch. Click the 'Pull Request' button and fill out the form.
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
Go to https://github.com/yourusername/cozy-papillon and select your branch. Click the 'Pull Request' button and fill out the form.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -116,12 +116,12 @@ You can help us by making our community even more vibrant. For example, you can
 
 
 
-[issues]: https://github.com/cozy/ogma/issues/new
+[issues]: https://github.com/cozy/Papillon/issues/new
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
[issues]: https://github.com/cozy/cozy-papillon/issues/new
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -116,12 +116,12 @@ You can help us by making our community even more vibrant. For example, you can
...
 [pr]: https://help.github.com/categories/collaborating-with-issues-and-pull-requests/
 [forking]: http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html
 [stdjs]: http://standardjs.com/
 [commitmsg]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
-[localization]: https://github.com/cozy/ogma/blob/master/README.md#localization
+[localization]: https://github.com/cozy/Papillon/blob/master/README.md#localization
```
Comment (cballevre) [inline CONTRIBUTING.md]:
```suggestion
[localization]: https://github.com/cozy/cozy-papillon/blob/master/README.md#localization
```

**Review by cballevre — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (paultranvan) [reply CONTRIBUTING.md]: Uh, I should have read this more carefully. Thank you for you vigilance!

**Pull request merged**